src/components/Results.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import moment from "moment";
+
+const Results = (props) => {
+  const res = props.results;
```

Comment:
Not sure if this intermediate value is needed? Could just do `props.results.map()`